WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -285,6 +285,7 @@ public GridViewHolder(View view) {
 
             videoOverlayContainer = (ViewGroup) view.findViewById(R.id.frame_video_overlay);
             selectionCountContainer = (ViewGroup) view.findViewById(R.id.frame_selection_count);
+            selectionCountContainer.setVisibility(mBrowserType.isPicker() ? View.VISIBLE : View.GONE);
```

Comment:
This effectively fixes the issue.  I think we can get the same result by setting the visibility in the `onBindViewHolder` method [here](https://github.com/jorgecasariego/WordPress-Android/blob/568682e2ec147ce450dbd6a894a42f79ae9a42dd/WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java#L188) and [here](https://github.com/jorgecasariego/WordPress-Android/blob/568682e2ec147ce450dbd6a894a42f79ae9a42dd/WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java#L197).  The advantage of that would be to have all the visibility logic in one place.

Reply:
Thanks @theck13  

You're right! It is a lot better to do it all in onBindViewHolder